backend/server/server.go

```diff
@@ -201,9 +203,12 @@ func apiSubmitHandler(ctx context.Context, w http.ResponseWriter, r *http.Reques
 	if GLOBAL_STATSD != nil {
 		GLOBAL_STATSD.Count("hishtory.submit", int64(len(devices)), []string{}, 1.0)
 	}
+
+	w.WriteHeader(http.StatusNoContent)
```

Comment:
While I agree this is a reasonable use for a 204 response code, the hishtory client actually checks that the response code is 200 to detect a successful response (see [here](https://github.com/ddworken/hishtory/blob/efa9ddd6df236a7877b3c42921c3b6ac32f4bbcc/client/lib/lib.go#L647)). It would be better if the hishtory client was slightly more lenient (accepting any 2xx response code), but sadly we're always going to have some out-of-date hishtory clients that would error on a 204 response code. So I think we should skip this part of the change.